server/src/main/java/com/autoback/autoback/api/GitRepoService.java

```diff
@@ -38,29 +38,38 @@ public Optional<LinkConstruct> createAccessLinks(PATConstruct patRequest) {
             return Optional.empty();
         }
 
+        // we check if a (GitRepo) object already exists by that repolink
```

Comment:
@evilpoke

regarding code style-changes in the last commit, I don't like to do grafit-style comments.
If code is self-explainatory, it should not have comments.

Comments are for the "why", not the what/how.
For what/how we have code